sdk/include/aditof/camera_definitions.h

```diff
@@ -59,6 +59,11 @@ struct CameraDetails {
      * @brief The range of the camera in mm for the operating mode
      */
     int range;
+
+    /**
+     * @brief The number of bits used for ADC
```

Comment:
On a second thought, maybe not mention the ADC. Sorry.
What about: "The number of bits used for representing one pixel data." ?

Reply:
solved. thanks